libs/components/src/lib/functional-header/stories/vl-functional-header.stories.ts

```diff
@@ -113,3 +115,27 @@ FunctionalHeaderTabs.storyName = 'vl-functional-header - tabs';
 FunctionalHeaderTabs.args = {
     title: 'School- en studietoelagen',
 };
+
+export const FunctionalHeaderBreadcrumb = story(
+    functionalHeaderArgs,
+    ({ fullWidth, marginBottom, title, link }) => html`
+        <vl-functional-header
+            ?data-vl-full-width=${fullWidth}
+            data-vl-link=${link}
+            data-vl-margin-bottom=${marginBottom}
+            data-vl-title=${title}
+            data-vl-hide-back-link
```

Comment:
ik vermoed omdat het niet reactief is, dat je het hier niet aanpasbaar maakt, maar ik vind het juist heel relevant voor de afnemer dat ze dit hier kunnen aanpassen - ook al is het niet reactief

```html
data-vl-hide-back-link="${hideBackLink}"
```

Reply:
Ik heb het aanpasbaar gemaakt in de andere stories, ik had gekozen om dat niet te doen voor de breadcrumb story omdat een breadcrumb in combinatie met een terug-link geen valid use-case is.